unitylibs/core/workflow/workflow-firefly/action-binder.js

```diff
@@ -179,6 +239,14 @@ export default class ActionBinder {
       });
     }
     if (!this.query) this.query = this.inputField.value.trim();
+    if (this.inputField.value.length === 0) {
+      this.serviceHandler.showErrorToast({ errorToastEl: this.errorToastEl, errorType: '.icon-error-empty-input' }, 'Empty input');
+      return;
+    }
+    if (this.inputField.value.length > 750) {
+      this.serviceHandler.showErrorToast({ errorToastEl: this.errorToastEl, errorType: '.icon-error-max-length' }, 'Empty input');
```

Comment:
I believe a different error message should be used here.

Reply:
This message is authorable:
![image](https://github.com/user-attachments/assets/d9b0d100-dfc3-4f67-87ab-3c1d8244b26d)

Reply:
Thanks for updating, looks good!